src/components/ExpandableCard.js

```diff
@@ -96,7 +106,10 @@ const ExpandableCard = ({ children, contentPreview, title }) => {
     <Card>
       <Content>
         <Question>
-          <Title>{title}</Title>
+          <Header>
+            {svg && <img src={svg} />}
+            <Title>{title}</Title>
+          </Header>
           <TextPreview>{contentPreview}</TextPreview>
         </Question>
         <ButtonContainer onClick={() => setIsVisible(!isVisible)}>
```

Comment:
```suggestion
        <ButtonContainer>
```

Reply:
If we agree on this change we also need to add `pointer: cursor` to the `Card` styled-component within ExpandableCard.js

Reply:
Only hesitation with doing this... It's nice for opening/closing, but if you want to select any of the expanded text (at least on desktop with a mouse), it will close the card when you un-click to finish your selection.

I tried it by moving `onClick={() => setIsVisible(!isVisible)}` to `<Content>` instead, which works pretty well by only closing when you click the top section of the card, but it does leave the padding un-clickable. Curious your thought on this.

Reply:
Tried this out... not sure I love it enough to keep it in for this PR, so reverted it back. The "ExpandableCard" is an existing component, which if we'd like to revamp it I have no problem with that, but requesting we address that separately to limit scope creep here.

Reply:
Agreed. As I noted above, this is why I think we should address that separately.